build/buildorder.props

```diff
@@ -12,24 +12,24 @@
     <RepositoryBuildOrder Include="BasicMiddleware" Order="9" />
     <RepositoryBuildOrder Include="Antiforgery" Order="10" />
     <RepositoryBuildOrder Include="IISIntegration" Order="10" RootPath="$(RepositoryRoot)src\IISIntegration\" />
-    <RepositoryBuildOrder Include="StaticFiles" Order="11" />
-    <RepositoryBuildOrder Include="ResponseCaching" Order="11" />
-    <RepositoryBuildOrder Include="Session" Order="11" />
-    <RepositoryBuildOrder Include="ServerTests" Order="11" />
     <RepositoryBuildOrder Include="CORS" Order="12" />
+    <RepositoryBuildOrder Include="StaticFiles" Order="11" RootPath="$(RepositoryRoot)src\StaticFiles\" />
```

Comment:
Can you restore the order of this file? MSBuild doesn't actually sort numerically, so things are building out of order now.

```
 10. Antiforgery, IISIntegration
 12. CORS, Routing, Diagnostics  
 11. StaticFiles, ResponseCaching, Session, ServerTests
```